src/hardware/vga_draw.cpp

```diff
@@ -2054,6 +2054,13 @@ ImageInfo setup_drawing()
 		return pcjr_or_tga();
 	};
 
+	// As far as I can tell, all Tandy modes have a height of 200.
+	// Some games (ex. Impossible Mission II) fiddle with vga.other.vdend
+	// The result of this should be rendering a short (by height) image in the horizonal center
+	// With black on the top/bottom
+	// Use this hard-coded value when calculating pixel aspect ratio so this effect looks correct.
+	constexpr uint16_t TandyAspectHeight = 200;
```

Comment:
Given we need to fix the composite paths too, and those composite modes affect CGA too, maybe call this `CgaTandyAspectHeight`? Well, PCjr would be affected too, but then the name would be even sillier...

Reply:
Yep, good catch.  Applied the change to the composite modes and confirmed it looks correct there as well.